packages/app/cypress/e2e/runner/reporter-ct-mount-hover.cy.ts

```diff
@@ -0,0 +1,51 @@
+import type { fixtureDirs } from '@tooling/system-tests'
+
+type ProjectDirs = typeof fixtureDirs
+
+const PROJECTS: {projectName: ProjectDirs[number], test: string}[] = [
+  { projectName: 'angular-14', test: 'app.component' },
+  { projectName: 'vueclivue2-configured', test: 'HelloWorld.cy' },
+  { projectName: 'react-vite-ts-configured', test: 'App.cy' },
+  { projectName: 'react18', test: 'App.cy' },
+  { projectName: 'create-react-app-configured', test: 'App.cy' },
+  { projectName: 'vueclivue3-configured', test: 'HelloWorld.cy' },
+  { projectName: 'nuxtjs-vue2-configured', test: 'Tutorial.cy' },
+]
+
+for (const { projectName, test } of PROJECTS) {
```

Comment:
Tests are very thorough, and I know this is already :heavy_check_mark: , but I wonder if we want to consider adding these to an existing test in the future? Scaffolding all these projects is pretty expensive (in terms of CI time) for something that could be exercised in one of the other tests we've got for the various frameworks.